src/test/data/JsonSerializableTutorsPetTest/typicalStudentsTutorsPet.json

```diff
@@ -1,36 +1,43 @@
 {
   "_comment": "Tutor's Pet save file which contains the same Student values as in TypicalStudents#getTypicalTutorsPet()",
```

Comment:
It would be good to edit the constructors in the `TypicalStudents` file as well to ensure the test constants are truly identical.

Reply:
Resolved by specifically adding Uuids into TypicalStudents